guildwars2/wallet.py

```diff
@@ -20,6 +21,54 @@ class WalletMixin:
                 pass
             await ctx.send_help(ctx.command)
 
+    def get_emoji_string(self, ctx, name):
+        emoji_string = name.replace(" ", "_").lower()
+        emoji_string = emoji_string.replace("'", "")
+        return self.get_emoji(ctx, emoji_string)
```

Comment:
get_emoji() already does the first part, though what's that part for?
```
emoji_string.replace("'", "")
```

Reply:
this is because of ' in Trader's Key, discord doesnt allow ' in emoji names

Reply:
Hmm, just modify get_emoji then if it's an illegal character.

Reply:
Modified get_emoji and removed it from wallet.py